pipelines/rj_cor__precipitacao_alertario_sftp/tasks.py

```
@@ -0,0 +1,448 @@
+        table_id,
+    )
+
+    try:
+        client = bigquery.Client(project=project_id)
+
+        query = f"""
+        SELECT
+            MAX(data_medicacao) as max_data_medicacao
+        FROM
+            `{project_id}.{dataset_id}.{table_id}`
+        """
```

Ruff (0.16.1)

[error] Possible SQL injection vector through string-based query construction

(S608)